package.json

```diff
@@ -38,15 +39,16 @@
     "htmlparser": "^1.7.7",
     "jed": "^1.1.1",
     "mini-css-extract-plugin": "^0.11.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "po2json": "^1.0.0-alpha",
```

Comment:
you can probably replace that with `gettext-parser`: the cockpit po plugin doesn't use po2json anymore

Reply:
skobyda/cockpit-certificates#58 and cockpit-project/cockpit-podman#731